Makefile

```diff
@@ -208,3 +208,8 @@ clean-snapshot:
 .PHONY: clean-dist
 clean-dist:
 	rm -rf $(DISTDIR) $(TEMPDIR)/goreleaser.yaml
+
+.PHONY: bootstrap-skaffold
+bootstrap-skaffold:
+	$(call title, Cloning chart for local skaffold dev)
+	git clone git@github.com:anchore/anchore-charts.git
```

Comment:
Just realized, should this use the https endpoint?

Reply:
It can do yeah especially given that's a public repo that doesn't need creds to clone anyway. I'll change it quickly before merging.